coe-cli/src/commands/devops.ts

```diff
@@ -456,106 +465,109 @@ class DevOpsCommand {
         }
     }
 
-    async createMakersBuildVariables(args: DevOpsInstallArguments, connection: azdev.WebApi, securityContext: DevOpsProjectSecurityContext) {
-        connection = await this.createConnectionIfExists(args, connection)
+    async createMakersBuildVariables(args: DevOpsInstallArguments, connection: azdev.WebApi, securityContext: DevOpsProjectSecurityContext) 
+    {
+        let projects = [args.projectName, args.pipelineProjectName]
```

Comment:
@mikefactorial need to check if args.projectName and args.pipelineProjectName are the same.